### PR TITLE
Added new screens

### PR DESCRIPTION
**Pull request by chetan-jain-atos**

### CODE REVIEW

File: app/views/offshore/TaxBeforeFiveYearsView.scala.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
+        <h1 class="govuk-heading-xl">@messages("taxBeforeFiveYears.heading", year)</h1>
+
+        @hmrcCharacterCount(
+            CharacterCount(
+                id = "taxBeforeFiveYears",
+                name = "taxBeforeFiveYears",
+                maxLength = Some(500),
```
Comment (burns238) [inline app/views/offshore/TaxBeforeFiveYearsView.scala.html:47]: I think the max length needs to be 5000, what does the ticket say?

File: app/forms/offshore/TaxBeforeSevenYearsFormProvider.scala [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import play.api.data.Form
+import play.api.data.Forms._
+import models.TaxBeforeSevenYears
+
+class TaxBeforeSevenYearsFormProvider @Inject() extends Mappings {
+
+   def apply(year: String): Form[TaxBeforeSevenYears] = Form(
+     mapping(
+      "taxBeforeSevenYears" -> text("taxBeforeSevenYears.error.taxBeforeSevenYears.required", Seq(year))
+        .verifying(maxLength(500, "taxBeforeSevenYears.error.taxBeforeSevenYears.length"))
```
Comment (burns238) [inline app/forms/offshore/TaxBeforeSevenYearsFormProvider.scala:31]: I think it needs to be max length 5000

File: app/views/offshore/TaxBeforeFiveYearsView.scala.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+@layout(pageTitle = title(form, messages("taxBeforeFiveYears.title", year))) {
+
+    @formHelper(action = controllers.offshore.routes.TaxBeforeFiveYearsController.onSubmit(mode), 'autoComplete -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
+        <h1 class="govuk-heading-xl">@messages("taxBeforeFiveYears.heading", year)</h1>
```
[the remark this answers is not in the document]
Comment (chetan-jain-atos) [reply app/views/offshore/TaxBeforeFiveYearsView.scala.html]: Fixed!!

File: app/views/offshore/TaxBeforeFiveYearsView.scala.html (hunk 1 above)
Comment (chetan-jain-atos) [reply app/views/offshore/TaxBeforeFiveYearsView.scala.html]: Fixed!!

File: app/pages/offshore/TaxBeforeFiveYearsPage.scala [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages
+
+import models.TaxBeforeFiveYears
+import play.api.libs.json.JsPath
+
+case object TaxBeforeFiveYearsPage extends QuestionPage[TaxBeforeFiveYears] {
```
[the remark this answers is not in the document]
Comment (chetan-jain-atos) [reply app/pages/offshore/TaxBeforeFiveYearsPage.scala]: Fixed!!

File: app/forms/offshore/TaxBeforeSevenYearsFormProvider.scala (hunk 2 above)
Comment (chetan-jain-atos) [reply app/forms/offshore/TaxBeforeSevenYearsFormProvider.scala]: Fixed!!

**Review by msatta — APPROVED** [review]

**Pull request merged**